features/0023-did-exchange/README.md

```diff
@@ -286,8 +148,8 @@ The _invitee_ will provision a new DID according to the DID method spec. For a P
 
 An invitation is presented in one of two forms:
 
-* An explicit invitation with its own `@id` as seen in the three examples [above](#invitation-message-with-public-invitation-did).
-* An [implicit](#implicit-invitation) invitation contained in a DID document's [`service`](https://w3c-ccg.github.io/did-spec/#service-endpoints) attribute.
+* An explicit out-of-band invitation or request with its own `@id`.
```

Comment:
Should clarify that this is "or a DID Exchange `request`", and that it is not an "oob-request" message.  Reading this one could be confused.

Reply:
Remove reference to the OOB request message.